sql/core/src/main/scala/org/apache/spark/sql/execution/subquery.scala

```diff
@@ -194,6 +272,19 @@ case class PlanSubqueries(sparkSession: SparkSession) extends Rule[SparkPlan] {
         }
         val executedPlan = new QueryExecution(sparkSession, query).executedPlan
         InSubqueryExec(expr, SubqueryExec(s"subquery#${exprId.id}", executedPlan), exprId)
+      case expressions.Exists(sub, children, exprId) =>
+        val expr = if (children.length == 1) {
+          children.head
+        } else {
+          CreateNamedStruct(
+            children.zipWithIndex.flatMap { case (v, index) =>
+              Seq(Literal(s"col_$index"), v)
+            }
+          )
+        }
+        val executedPlan = new QueryExecution(sparkSession, Project(Nil, sub)).executedPlan
+        ExistsExec(expr, sub.treeString,
```

Comment:
Why `ExistsExec` needs a `expr` parameter?

Reply:
> Why `ExistsExec` needs a `expr` parameter?

Remove `expr` and `subQuery`